packages/cli/src/util/deploy/get-deployment-by-id-or-host.ts

```diff
@@ -11,12 +11,30 @@ import mapCertError from '../certs/map-cert-error';
 
 type APIVersion = 'v5' | 'v10';
 
+export type GetDeploymentByIdOrHostReturnType =
+  | DeploymentNotFound
+  | DeploymentPermissionDenied
+  | InvalidDeploymentId
+  | ReturnType<typeof mapCertError>;
+
 export default async function getDeploymentByIdOrHost(
   client: Client,
   contextName: string,
   idOrHost: string,
-  apiVersion: APIVersion = 'v5'
-) {
+  apiVersion: 'v5'
+): Promise<DeploymentV5 | GetDeploymentByIdOrHostReturnType>;
+export default async function getDeploymentByIdOrHost(
+  client: Client,
+  contextName: string,
+  idOrHost: string,
+  apiVersion: 'v10'
```

Comment:
Any reason why you're using v10 instead of the latest (v13)?

https://vercel.com/docs/rest-api#endpoints/deployments/get-a-deployment-by-id-or-url

Reply:
Good catch! I thought v10 was the latest. I have no idea how I came to that conclusion. v13 has been the latest for almost 2 years now. Wow. I'll update to v13 and update the type accordingly. Thanks!

Reply:
Ah, this is where I got v10 from: https://github.com/vercel/vercel/blob/main/packages/cli/src/util/deploy/get-deployment-by-id-or-host.ts#L12